src/symbol/glyph_source.js

```diff
@@ -5,6 +5,9 @@ const verticalizePunctuation = require('../util/verticalize_punctuation');
 const Glyphs = require('../util/glyphs');
 const GlyphAtlas = require('../symbol/glyph_atlas');
 const Protobuf = require('pbf');
+const TinySDF = require('@mapbox/tiny-sdf');
+const isChar = require('../util/is_char_in_unicode_block');
+/* eslint-disable new-cap */
```

Comment:
OK, moved the disabling to the two lines where it's necessary. It's not actually a `new lowercaseThing()` but the converse that's triggering the warning (a function starting with an uppercase letter on account of the function name being based on the Unicode block name).